### PR TITLE
velocity updates

### PR DESCRIPTION
**Pull request by jmhauck**

### CODE REVIEW

File: packages/common/src/velocityHelpers.ts [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        const orgCapabilities = getProp(subscriptionInfo, "orgCapabilities");
+        /* istanbul ignore else */
+        if (Array.isArray(orgCapabilities)) {
+          orgCapabilities.some(c => {
+            velocityUrl = c.velocityUrl;
+            return velocityUrl;
+          });
+        }
+        // add the base url to the templateDictionary for reuse
+        templateDictionary.velocityUrl = velocityUrl;
```
Comment (MikeTschudi) [inline packages/common/src/velocityHelpers.ts:58]: Do you intend to set `templateDictionary.velocityUrl` to "" in the case where "orgCapabilities" is not returned?

**Review by MikeTschudi — APPROVED** [review]

File: packages/common/src/velocityHelpers.ts (hunk 1 above)
Comment (jmhauck) [reply packages/common/src/velocityHelpers.ts:58]: @MikeTschudi good catch...the "" will be ok but I do need to make some changes to properly avoid redundant and bad requests when the org doesn't support velocity.

Comment (jmhauck) [reply packages/common/src/velocityHelpers.ts:58]: @MikeTschudi we will prevent users from deploying solutions that have velocity items via an upcoming change to the app but I didn't handle it correctly for users of solution.js outside the app

**Comment by jmhauck** [comment]
#674

**Pull request merged**